modules/doc/content/framework_development/authorship_guidelines.md

```diff
@@ -0,0 +1,73 @@
+of the MOOSE framework paper representative. To do that, we publish a summary of the new developments in the framework
+and use the list of the principal contributors since the previous paper as the list of authors for the new paper.
+
+The contributors are listed in two categories.
+
+- Semi-permanent contributors are included in every new MOOSE framework paper by default. These contributors
+  have written a very large part of the MOOSE framework and must be credited, at least until their contributions
+  are overtaken by others.
+
+The current criteria for semi-permanent authorship are listed below:
+
+- Top 10 line of codes in the `framework/` and other utility folders or
+- Top 10 commits in the `framework/` folder
+
+
+!alert note
+While it seems poor practice to recognize contribution by LOC or commit numbers, this criteria did not
+exist when these contributions were made, so the metric did not influence the coding style of these authors.
+These criteria may be reformulated in the future in terms of features.
+
+
+- Transient contributors are included in the next MOOSE framework paper for the period(s) in which they contributed.
+  These contributors have played an important role in MOOSE development since the previous paper was published.
+
+
+The current guidelines for transient authorship are listed below:
+
+- 1 major contribution or
+- 4 minor contributions and significant involvement in MOOSE-related activities or
+- 10 fixup contributions and significant involvement in MOOSE-related activities
+
+
+Contributions are only tallied for the *framework and supporting utilities*. Modules are deliberately excluded
+as developers of a module should publish a module paper and report on new developments outside of the MOOSE framework paper.
+MOOSE-related activities are developments in the MOOSE modules and MOOSE applications.
+
+A list of major, minor and fixup example contributions has been developed as guidelines for
+classification and can be shared upon request. The classification of contributions is decided when
+gathering the list of authors for the next paper.
+
+!alert note
+Contributors who do not meet the criteria for authorship in the paper will see their contributions from
+the past period considered for the next paper.
+
+!alert note
+Code reviews and user support are currently not formalized in the criteria. Depending on the scale of the effort,
+we may consider them at the level of minor and fixup contributions.
+
+!alert note
+The contribution guidelines may be scaled depending on the time elapsed between successive publications. The current
+guidelines were developed for an 18 months time frame between two publications.
+
+## MOOSE Modules paper authorship guidelines
+
+Modules should strive to follow the same principles as the framework with regards to authorship inclusivity
+and representativity. However, most modules are not developed by the framework team and have their own policies.
```

Comment:
I would add something about contacting the member of the CCB related to a given module (with a link to the CODEOWNERS file perhaps?)

Reply:
Codeowner s file s pretty rough git no?

Reply:
A list of module directories next to GitHub handles? I don't think that's hard to parse for someone who is a contributor. Adding a link is just a suggestion, though.

Reply:
I dont think it matches each module's CCB that well. Reactor module for example, I would say most of their CCB is at ANL and not a codeowner because git still isnt developing this obvious need of non-maintainer codeowners